src/Catalog/Dnx/DnxCatalogCollector.cs

```diff
@@ -135,7 +141,12 @@ public class DnxCatalogCollector : CommitCollector
 
         private async Task ProcessPackageDeleteAsync(string id, string version, CancellationToken cancellationToken)
         {
-            await _dnxMaker.DeletePackage(id, version, cancellationToken);
+            await _dnxMaker.DeletePackageAsync(id, version, cancellationToken);
+
+            await _dnxMaker.UpdatePackageVersionIndexAsync(
```

Comment:
The order of operations changed from the previous implementation(previously it was Update metadata followed by the  nuspec, nupkg deletes now the update metadata is the last). This could lead to cases when the nupkg is deleted and the metadata is not.

Reply:
Good catch.